packages/haproxy/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,14 +1,9 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -14,7 +14,11 @@
-   name: Baptiste Assmann
- - email: dkorunic@haproxy.com
-   name: Dinko Korunic
--name: kubernetes-ingress
-+name: haproxy
```

Comment:
Everything in this PR is correct related to the package itself is correct, but you didn't change the name back to `haproxy` when you updated the chart and this resulted in a new kubernetes-ingress chart along with the old haproxy chart. You need to rebase your first commit to add this change (Using the make commands, make prepare, apply the change, make patch, make clean, then amend commit). Make sure you delete your `make charts` commit (2nd commit), re-run it and re-commit the output so the index.yaml is updated correctly.

![image](https://user-images.githubusercontent.com/8951271/134424632-898d19bc-56c2-44ae-a73d-c13ce9a7674e.png)

CC @alex-isv

Reply:
Sorry about that.

I've rebased and re-done the change and then force-pushed to my branch so that should be sorted out now.